uqcs/templates/form.mako

```diff
@@ -65,9 +65,14 @@
           type="email"
           class="form-control"
           id="emailInput"
-          placeholder="john@example.com"
+          placeholder=""
           required="required"
         />
+        <p class="form-text text-muted">Valid UQ emails are: 
```

Comment:
Should specify that these are valid student email formats.

Not sure about the "xx" bits. I think people can work this bit out if they get the domain right.

Reply:
Could you just put the appropriate formats in the placeholder somehow?

Reply:
Something like "Valid UQ email formats are"?

I'll get rid of xx, UQ's website had it as a placeholder for numbers I think.

Placeholder is difficult since there are 3 different formats and we need to support other domains too. Something [like this](https://getbootstrap.com/docs/4.0/components/input-group/#basic-example) would be nice, but I don't know how we can work it in.

Reply:
I'd say leave the xx bit in. Doesn't cause any harm for those who know of it, but do help those could forget

Reply:
Fwiw, very few people have a number in their email. I'd imagine those that do remember it.